### PR TITLE
Travis-CI: Add tests to Docker images

### PR DESCRIPTION
**Pull request by yukiisbored**
This Travis configuration tests building the docker image and building
a Python project with the docker image.

### CODE REVIEW

**Comment by yukiisbored** [comment]
Here's the Travis Build: https://travis-ci.org/yukiisbored/rultor-python/builds/189074215

**Comment by jayvdb** [comment]
Installing coala by itself isnt terribly useful.

A proper test it would need to run coala on the coala & coala-bears source, with the minimum dependencies installed to do what those `.coafile` request.

**Comment by jayvdb** [comment]
Failing on https://travis-ci.org/yukiisbored/rultor-python

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+     cd coala &&
+     pip3 install -r requirements.txt &&
+     python3 setup.py bdist_wheel &&
+     pip3 install ./dist/coala-*.whl &&
+     cd .. &&
+     git clone https://github.com/coala/coala-bears &&
+     cd coala-bears &&
+     pip3 install -r requirements.txt &&
+     python3 setup.py bdist_wheel &&
+     pip3 install $(ls ./dist/*.whl)\"[alldeps]\" &&
```
Comment (jayvdb) [inline .travis.yml]: maybe try printing out `$(ls ./dist/*.whl)` and removing `\"[alldeps]\"` -- the latter is part of a conversion which hasnt happened yet - you may need to manually install `language_check` , as that is the main difference.

Comment (yukiisbored) [reply .travis.yml]: It looks like `language-check` is part of the docker image. https://travis-ci.org/yukiisbored/rultor-python/builds/192384268#L306

**Comment by yukiisbored** [comment]
@jayvdb I noticed `setup.py bdist_wheel` didn't create a wheel file, is this an issue with coala-bears? https://travis-ci.org/yukiisbored/rultor-python/builds/192386649#L1756

**Comment by jayvdb** [comment]
No, that isnt a problem with coala-bears : https://travis-ci.org/jayvdb/coala-bears/jobs/192418945#L4979
IMO, for now this image testing can just install normally without a wheel, and we worry about the wheel later.

**Comment by yukiisbored** [comment]
@jayvdb Green build: https://travis-ci.org/yukiisbored/rultor-python/builds/197308184

**Comment by yukiisbored** [comment]
ah, it looks like I forgot to install ntlk data and csslint

**Pull request merged**